### DISCUSSION TITLE
Getting a 502

### DISCUSSION THREAD

**Discussion by kenbartlett** [FunkeyFlo/ps5-mqtt#363] [Q&A]
I just got the home assistant green and added the PS5 add on. The app seems to start fine but when I click “open web ui” I get a 502. Anyone else run into this?

![IMG_0242](https://github.com/FunkeyFlo/ps5-mqtt/assets/117783795/4c1c296e-ff40-4763-a864-8629e850e6d3)

**Comment by nexusis7** [comment]
Same. not working anymore.. 2024.1.2

**Comment by FunkeyFlo** [comment]
Seems to be on your side. As it's working fine here on the latest version. (See screenshot) 

Are you using a reverse proxy? Maybe a configuration issue there that's been triggered by an update?

![Screenshot_20240107-093111](https://github.com/FunkeyFlo/ps5-mqtt/assets/4623715/da682a5a-2e92-46ef-8e56-111760a2e230)

**Reply by kenbartlett** [reply]
When I installed it I didn't touch any settings. How would I identify if I'm using a reverse proxy or configurations aren't correct?

**Comment by nexusis7** [comment]
> Seems to be on your side. As it's working fine here on the latest version. (See screenshot)
> 
> Are you using a reverse proxy? Maybe a configuration issue there that's been triggered by an update?
> 
> ![Screenshot_20240107-093111](https://private-user-images.githubusercontent.com/4623715/294746761-da682a5a-2e92-46ef-8e56-111760a2e230.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ2NTQ1ODksIm5iZiI6MTcwNDY1NDI4OSwicGF0aCI6Ii80NjIzNzE1LzI5NDc0Njc2MS1kYTY4MmE1YS0yZTkyLTQ2ZWYtOGU1Ni0xMTE3NjBhMmUyMzAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMDdUMTkwNDQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Njg4YmJlOGVmYzAwMTM2YTRjY2MzOTdmNWQ5NGVlNjhlMTcyMWQ4ZGY2YTI3N2VmZmFkMDk1Y2QxMDlmNjliYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.AD89nKI_Ol76Yi9TZ_zmo_NxM2TIyd-Yl41pF8syXrQ)

i get this page but nothing ever loads up.. doesn't even give me the screen to link up my ps5 again when i reinstall. i removed everything and re installed mqtt as well. i just get this page with the refresh device button

**Comment by bastelbert** [comment]
I've got the same issue. Fresh installation, didn't touch the config yet. 
Core 2024.1.6
Supervisor 2024.01.1
Operating System 11.5
Frontend 20240104.0

**Comment by FunkeyFlo** [comment]
Have you tried turning it off and on again? (HA)

**Reply by bastelbert** [reply]
I've seen the IT Crowd a hundred times, so yes, I've restarted everything! ;)

**Reply by bastelbert** [reply]
Problem solved. 
I had to add a dedicated user to the MQTT plugin  config:

logins:
  - username: mqttuser
    password: xxxxxxxxx
